modules/rhcos-match-vulnerability.adoc

```diff
@@ -6,6 +6,6 @@
 = Vulnerability matching
 
 [role="_abstract"]
-Central services, which include Central and Scanner, perform vulnerability matching. Scanner uses Red{nbsp}Hat's Open Vulnerability and Assessment Language (OVAL) v2 security data streams to match vulnerabilities on {op-system-first} software components.
+Central services, which include Central and Scanner, perform vulnerability matching. The StackRox Scanner uses Red{nbsp}Hat's Open Vulnerability and Assessment Language (OVAL) v2 security data streams to match vulnerabilities on {op-system-first} software components. Scanner V4 uses https://security.access.redhat.com/data/csaf/v2/vex/[VEX].
 
-Unlike the earlier versions, {product-title-short} 4.0 no longer uses the Kubernetes node metadata to find the kernel and container runtime versions. Instead, it uses the installed {op-system} RPMs to assess that information.
+Unlike the earlier versions, {product-title-short} releases after 4.0 no longer use the Kubernetes node metadata to find the kernel and container runtime versions. Instead, {product-title-short} uses the installed {op-system} RPMs to assess that information.
```

Comment:
this was in 4.0? why did i think it was earlier? doesn't seem like the version was changed in this PR, so I guess that's true (just didn't remember)

Reply:
Looks like the PR for the doc changes was done for 4.0 so I think this is correct.